### DISCUSSION TITLE
Add an option for format the Pit value in the RangeSlider

### DISCUSSION THREAD

**Discussion by samouss** [algolia/instantsearch#5207] [Ideas]
**Do you want to request a *feature* or report a *bug*?**

Feature.

**Feature: What is your use case for such a feature?**

With the current implementation of the Slider we can format the tooltip value but not the Pit value. In some cases the computed value has a decimal but it's not always relevant (e.g. year).

![untitled](https://user-images.githubusercontent.com/6513513/36793468-c87a742c-1c9d-11e8-86f4-da1aff0802d4.png)

**Feature: What is your proposed API entry? The new option to add? What is the behavior?**

Add a similar option to [`RangeSliderTooltipOptions`](https://community.algolia.com/instantsearch.js/v2/widgets/rangeSlider.html#struct-RangeSliderTooltipOptions) for the Pit value.

**Comment by bobylito** [comment]
Sounds reasonable given that we already have an option for the tooltip.
